src/mmrelay/cli.py

```
@@ -1438,32 +1585,164 @@ def main() -> int:
+    elif args.command == "doctor":
+        return handle_doctor_command(args)
+    elif args.command == "verify-migration":
+        return handle_verify_migration_command(args)
+    elif args.command == "migrate":
+        return handle_migrate_command(args)
     else:
         print(f"Unknown command: {args.command}")
         return 1
 
 
+def handle_config_paths(args: argparse.Namespace) -> int:
```

Ruff (0.14.14)

[warning] Unused function argument: `args`

(ARG001)